README.md

```diff
@@ -68,16 +61,9 @@ profiles of the SPDX model.
 
 #### Dataset
 
-The Dataset profile describes the characteristics of the dataset(s) used by the AI system.
-Fields include type of the dataset (image, text, relational DB, …), how the data is collected,
-how the collected data would be used, size of the dataset, information about noise in the
-fields of the dataset and/or noise that affects the entire dataset, information about
-preprocessing before dataset is constructed, sensors used to collect data, known biases
-in the data of the dataset, information about whether the dataset contains sensitive or
-identifiable personal information, anonymization methods used to mask sensitive or identifiable
-personal information, confidentiality levels of data points in the dataset as defined in the
-Traffic Light Protocol, information about dataset update mechanism, and information about
-availability aspects of the dataset (public, restricted, …).
+The Dataset profile describes a dataset's core aspects (type, size, collection
+method) and its preparation for AI use (preprocessing, noise handling, data
```

Comment:
Dataset can be used for more than just AI,  so we may want to be careful on this part.

Reply:
Agree. I will adjust it.